torch/_subclasses/fake_tensor.py

```diff
@@ -31,6 +39,18 @@
 
 CONSTANT_NUMEL_LIMIT = 1
 
+CNT = 0
+
+
+class Increment:
```

Comment:
That looks shady. Is it just for debugging?

Reply:
It's for the debug logging. I can delete the debug logging code but, IDK, I thought it was pretty useful, I feel like we should at least let it hang out in master a bit.

Reply:
I don't think this kind of logging is useful in general tbh. it just ends up bloating the code and generate so much lines that it is impossible to find anything in there.

But Dynamo and Inductor are already going down that path so I won't actively block this.

Reply:
Increment is a bit misleading. It's really something more like StackCount right, or RecursiveCount ?